x-pack/test/security_solution_cypress/cypress/e2e/investigations/alerts/detection_page_filters.cy.ts

```diff
@@ -117,19 +115,17 @@ describe.skip(`Detections : Page Filters`, { tags: ['@ess', '@serverless'] }, ()
     waitForAlerts();
   });
 
-  it('Default page filters are populated when nothing is provided in the URL', () => {
+  it('should populate page filters with default values when nothing is provided in the URL', () => {
     assertFilterControlsWithFilterObject();
   });
 
   context('Alert Page Filters Customization ', () => {
     it('should be able to delete Controls', () => {
```

Comment:
looking at this test again, I feel like it does not provide as much value as it could:
- first I think we should verify the length of controls before deleting one so we have something to compare before and after
- then is there a value in using `lt` (lower than) instead of an plain equal to 3 here?

Reply:
also looking a bit more at these first 3 tests (`should be able to delete Controls`, `should be able to add new Controls` and `should be able to edit Controls`) they could easily be merged into a single test:
- first start by deleting a control
- then add a new one back
- then edit one of the controls

It flows really well, we wouldn't have to reload the whole thing 3 times. The title `should be able to edit Controls` would work quite well! You could use 3 `cy.log` to describe each step of the test.
Just a thought...

Reply:
>first I think we should verify the length of controls before deleting one so we have something to compare before and after

Actually, we know that the `before` creates 4 controls so we do have something to compare.

>then is there a value in using lt (lower than) instead of an plain equal to 3 here?

All of below are okay in my opinion

```
< 4
!= 4
== 3
```

I have changed it to 3 for precision sake.

>also looking a bit more at these first 3 tests (should be able to delete Controls, should be able to add new Controls and should be able to edit Controls) they could easily be merged into a single test:

You have the right thinking. Initially, these were separate because we had `testIsolation: false` which make sure that there was no reload. But then we stop using that clause and these remained separate.

I have observed that complex tests create flakyness in the test so I try to test mostly one thing in the single test. But this seem to be simple enough so I have done this change.